tbx/people/models.py

```diff
@@ -82,6 +83,20 @@ def author_posts(self):
             .order_by("-date")
         ]
 
+    @property
+    def related_works(self):
+        # Get author
+        author_snippet = Author.objects.get(person_page__pk=self.pk)
+
+        # Get the latest 2 work pages by this author
+        works = (
+            WorkPage.objects.filter(authors__author=author_snippet)
```

Comment:
suggestion: 
```suggestion
            WorkPage.objects.filter(authors__author__person_page=self.pk)
```

should remove the need for `author_snippet = Author.objects.get(person_page__pk=self.pk)`